ops/terrad/terrad.go

```diff
@@ -27,92 +29,85 @@ const (
 	// deployed contract addresses
 	LINK
 	OCR2
+	RequesterAccessController
+	BillingAccessController
 )
 
-type Terrad struct {
-	url      string
-	chainID  string
-	keyID    string
-	addr     string
-	args     []string
-	Uploaded map[int]string
-	Deployed map[int]string
-}
-
 type key struct {
 	Name    string
 	Address string
 }
 
-func New(ctx *pulumi.Context) (Terrad, error) {
-	// check if terrad is installed
-	_, err := exec.LookPath("terrad")
+type Deployer struct {
```

Comment:
This should not be a migration/replacement but an addition as the first step.

For example, migrate `ops/terrad/terrad.go` as is to `ops/deployer/terrad.go` and add a separate `ops/deployer/guntlet.go`. Let's get both working first.